openmetadata-ui/src/main/resources/ui/src/constants/elasticsearch.constant.ts

```diff
@@ -77,3 +77,5 @@ export const RECREATE_INDEX_OPTIONS = [
     value: false,
   },
 ];
+
+export const ELASTIC_SEARCH_REINDEX = 'elasticSearchReindex';
```

Comment:
@Ashish8689 , we have `PipelineType`, so use that instead of creating the new constant.

Reply:
@Sachin-chaurasiya, also shouldn't we use localization instead of constants?

Reply:
@aniketkatkar97 , we don't need localization and constant here as we have an enum for `PipelineType` which is generated from schema.